src/Form/Resolver/Choice.php

```diff
@@ -105,7 +105,19 @@ private function getValues(array $field)
      */
     private function getYamlValues(Bag $field)
     {
-        $values = array_slice($field->get('values', []), 0, $field->get('limit'));
```

Comment:
You just need to add `true` as the 4th parameter to `array_slice()` to do natively what you're recreating below :wink: 

```
$values = array_slice($field->get('values', []), 0, $field->get('limit'), true);
```

Reply:
yay! 😀 
(actually, in such situations i dislike all those complex polymorphic functions)